support mutating scratch images

jordan-da opened the pull request:
closes https://github.com/google/go-containerregistry/issues/1881

Adds support for mutating `scratch` base images.

jordan-da commented:
Is this project not considering external contributions at this time?

mepi262 commented:
@thesayyn @martianoff cc @jordan-da 
Would you review this pull request?

mepi262 commented:
@jordan-da 
It seems that some checks are not working.
Would you try the following action?
https://stackoverflow.com/questions/52200096/github-pull-request-waiting-for-status-to-be-reported

jordan-da commented:
> It seems that some checks are not working.

sure

jordan-da commented:
> @jordan-da It seems that some checks are not working. Would you try the following action? https://stackoverflow.com/questions/52200096/github-pull-request-waiting-for-status-to-be-reported

blehhh... I accidentally pushed `origin` to my fork, which automatically closed the pr : /

I fixed the merge conflicts and rebased, I guess i need another approval now cc @martianoff 

Sorry for the mess folks

jordan-da commented:
> @jordan-da It seems that some checks are not working. Would you try the following action? https://stackoverflow.com/questions/52200096/github-pull-request-waiting-for-status-to-be-reported

@mepi262 i followed the steps in the post, force pushed an amended commit, this checks are still stuck.. do they require a lifecycle phase change?
